apps/web/src/components/Spaces/Common/Dropdown.tsx

```diff
@@ -0,0 +1,20 @@
+import { Menu } from '@headlessui/react';
+import type { FC, ReactNode } from 'react';
+import React from 'react';
+
+type DropdownProps = {
+  triggerChild: JSX.Element;
+  children: ReactNode;
+};
+
+const Dropdown: FC<DropdownProps> = ({ children, triggerChild }) => {
+  return (
+    <Menu>
+      <Menu.Button>{triggerChild}</Menu.Button>
+      <Menu.Items className="absolute">
```

Comment:
This is absolute to which relative element? @bigint do we not have dropdown?

Reply:
RN we dont have dropdown, just using the headless UI one everywhere, but I will add a comp for it asap